### PR TITLE
Fix login view

### PR DESCRIPTION
**Pull request by Luiz-FS**
**Feature/Bug description:** The login page in the mobile device browsers was broken.

screenshot from 2018-12-04 15-23-31

**Solution:** Replace the angular directives by grid layout, and adjust the size of the components.

screenshot from 2018-12-04 15-27-49

screenshot from 2018-12-04 15-27-05

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/app.js [hunk 1]
```diff
@@ -515,6 +515,6 @@
     }
 
     (function main() {
-        initServiceWorker();
+        //initServiceWorker();
```
Comment (ruanres) [inline frontend/app.js]: undo

File: frontend/auth/login.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
       
-      <md-card class="custom-card login-card">
-        <md-card-content layout="row" layout-align="center center">
+      <div class="login-card">
+        <div id="form-content">
           <div class="custom-card-content login-card-content">
             <img id="login-logo" class="custom-card-logo" src="/app/images/logowithname.png"/>
             
-            <form class="login-form" ng-submit="loginCtrl.loginWithEmailPassword()">
+            <form style="width: 100%;" class="login-form" ng-submit="loginCtrl.loginWithEmailPassword()">
```
Comment (ruanres) [inline frontend/auth/login.html]: remove style from template

File: frontend/auth/login.html [hunk 3]
```diff
@@ -36,7 +36,7 @@
 
             <md-divider></md-divider>
 
-            <p class="login-text">
+            <p class="login-text" style="margin-top: 4px;">
```
Comment (ruanres) [inline frontend/auth/login.html]: remove style from template

File: frontend/index.html [hunk 4]
```diff
@@ -54,7 +54,7 @@
 </head>
 <body ng-app="app" ng-cloak layout="column">
 
-  <div flex layout="column" ui-view="main"></div>
+  <div flex style="height: 100%;" layout="column" ui-view="main"></div>
```
Comment (ruanres) [inline frontend/index.html]: remove style from template

File: frontend/auth/login.html (hunk 3 above)
Comment (Luiz-FS) [reply frontend/auth/login.html]: Fixed.

File: frontend/index.html (hunk 4 above)
Comment (Luiz-FS) [reply frontend/index.html]: Fixed.

**Pull request merged**